tests/Unit/CssInlinerTest.php

```diff
@@ -773,14 +783,14 @@ public function matchedCssDataProvider()
      *
      * @dataProvider matchedCssDataProvider
      */
-    public function emogrifyAppliesCssToMatchingElements($css, $expectedHtml)
+    public function inlineCssAppliesCssToMatchingElements($css, $expectedHtml)
     {
         $cssDeclaration1 = 'color: red;';
         $cssDeclaration2 = 'text-align: left;';
         $subject = $this->buildDebugSubject(static::COMMON_TEST_HTML);
-        $subject->setCss(\sprintf($css, $cssDeclaration1, $cssDeclaration2));
+        $subject->inlineCss(\sprintf($css, $cssDeclaration1, $cssDeclaration2));
 
-        $result = $subject->emogrify();
+        $result = $subject->render();
 
```

Comment:
Now that `inlineCss()` performs the action being tested, the blank lines separating 'set-up', 'execute' and 'check result' no longer achieve that logical grouping (this also applies to many other tests below).

Reply:
Done.